setup.py

```diff
@@ -23,5 +23,9 @@
     ],
     install_requires=[
         'wagtail>=1.0.0',
+        'django-contrib-comments>=1.6.1',
+        'django-comments-xtd>=1.5.3',
```

Comment:
Both libs are imported at the top of `blog/management/commands/wordpress_to_wagtail.py`